### PR TITLE
Add Real Time Clock documentation

### PR DESCRIPTION
**Pull request by ethomas997**
Added 'Real Time Clock' doc

### CODE REVIEW

File: doc/Hardware Setup.md [hunk 1]
```diff
@@ -49,10 +49,14 @@ Complete wiring connections between each Arduino and the Raspberry Pi.
 
 Note: be sure all Receiver Nodes and the Raspberry Pi are tied to a common ground; if not, the i2c messages can be corrupted.
 ![system wiring](img/D5-i2c.png)
+A directional RF shield significantly improves the system's ability to reject false passes. See [RF shielding](Shielding%20and%20Course%20Position.md)
```
Comment (HazardCreative) [inline doc/Hardware Setup.md:52]: Duplicating this line intentional?

File: doc/Real Time Clock.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+# Real Time Clock
+
+The standard Raspberry Pi does not feature a real-time clock (RTC) -- when it has an internet connection the Pi will use it to fetch the current date and time, but if an internet connection is not available when the Pi starts up, its date/time will be wrong.  (It will usually revert to the last date/time it was using, and go from there.)
+
+Adding a real-time clock module addresses this problem.  The [DS3231](https://www.adafruit.com/product/3013) module is recommended, as it keeps very accurate time.  There are other, cheaper modules that will also do the job (albeit with less precision), such as the [PCF8523](https://www.adafruit.com/product/3295) and [DS1307](https://www.adafruit.com/product/3296)
```
Comment (HazardCreative) [inline doc/Real Time Clock.md:5]: I've used the DS1307. Its accuracy is not good at all.

File: doc/Real Time Clock.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+### Wiring
+
+The RTC module connects to the Raspberry Pi via the I2C bus.  The following four pads on the RTC module need to be wired to the corresponding connections going to the Pi:  `Vin, GND, SCL, SDA`.  If using the DS3231 or PCF8523, the Vin pad can be wired to 3.3V or 5V.  For the DS1307 use 5V.  The other pads (besides those four) on the RTC module should be left unconnected.
+
+Here is an example of wiring a DS3231 module via a Delta5 V2 PCB:
+
+![DS3231 wiring](img/RH_DS3231_D5PCB.jpg)
+
+If you're using one of the newer RotorHazard PCBs, there are several places on the board where these connections are available.
```
Comment (HazardCreative) [inline doc/Real Time Clock.md:17]: Link to PCB page

File: doc/Hardware Setup.md (hunk 1 above)
Comment (ethomas997) [reply doc/Hardware Setup.md:52]: Copy-paste error; thx.

File: doc/Real Time Clock.md (hunk 3 above)
Comment (ethomas997) [reply doc/Real Time Clock.md:17]: Sure

**Pull request merged**